### DISCUSSION TITLE
Unity crash when creating SOFA context

### DISCUSSION THREAD

**Discussion by 985-211** [InfinyTech3D/SofaUnity#76] [Help - Using SofaUnity] [answered]
I created a SOFA context in Unity, which causes the Unity editor to crash. What could be the reason for this? My license location is correct, and my license is valid This is my log[](url)
[Editor.log](https://github.com/InfinyTech3D/SofaUnity/files/14953521/Editor.log)

**Comment by epernod** [comment]
Hi,

are you sure this is the good log, I can't see any error related to Sofa-Unity.
Did you find what was the problem?

Best regards

**Reply by 985-211** [reply]
I first imported the SOFAUnity plugin into Unity, then replaced it with a valid license. Next, when I created the SOFA context in Unity, the Unity editor crashed. How should I resolve this issue? What type of logs do you need me to provide?

**Reply by epernod** [reply, marked as answer]
Hi,
normally the editor.log is the good one but I can't see any relevant error message in your.
Could you try to load one of the provided example like the examples/example_01_liver and check if it crashes as well.

Also could you explain the difference with your execution from the other discussion: https://github.com/InfinyTech3D/SofaUnity/discussions/75
